deploy/deploy-with-olm.yaml

```diff
@@ -10,9 +10,9 @@ kind: OperatorGroup
 metadata:
   name: local-operator-group
   namespace: local-storage
-spec:
-  targetNamespaces:
-  - openshift-storage
+  spec:
+    targetNamespaces:
+    - openshift-storage
```

Comment:
Strange. It only seems to work like this. Will need further investigation. Looks like the spec was included in the metada and was not taken into account when the yaml was made to work, so now enabling it breaks something. I'm not familiar enough with OLM abstractions to fix. So I'm reverting to the working state, but there's something wrong here.

OperatorGroup docs.
https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/operatorgroups.md